website/settings/defaults.py

```diff
@@ -2125,3 +2126,4 @@ def from_node_usage(cls,  usage_bytes, private_limit=None, public_limit=None):
 CAS_LOG_LEVEL = 3  # ERROR
 
 PREPRINT_METRICS_START_DATE = datetime.datetime(2019, 1, 1)
+DOMAIN_REGEX = re.compile('(?P<protocol>\w+://)?(?P<www>www\.)?(?P<domain>[\w-]+\.\w+)(?P<path>/\w*)?')
```

Comment:
drive-by: putting a regex in settings smells wrong -- might fit better as a constant defined close to the logic that uses it